packages/toolkit/src/view/model/view-model/ModelRunViews/Run.tsx

```diff
@@ -0,0 +1,209 @@
+import * as React from "react";
+import Link from "next/link";
+import { Model, Nullable } from "instill-sdk";
+
+import { Button, Form, Icons, TabMenu } from "@instill-ai/design-system";
+
+import {
+  CodeBlock,
+  ModelSectionHeader,
+  RunStateLabel,
+} from "../../../../components";
+import { defaultCodeSnippetStyles } from "../../../../constant";
+import {
+  convertToSecondsAndMilliseconds,
+  InstillStore,
+  useComponentOutputFields,
+  useInstillForm,
+  useInstillStore,
+  usePaginatedModelRuns,
+  useRouteInfo,
+  useShallow,
+} from "../../../../lib";
+import { getHumanReadableStringFromTime } from "../../../../server";
+import {
+  convertTaskNameToPayloadPropName,
+  convertValuesToString,
+  ModelOutputActiveView,
+} from "../ModelPlayground";
+
+export type ModelRunProps = {
+  id?: string;
+  model?: Model;
+};
+
+const selector = (store: InstillStore) => ({
+  accessToken: store.accessToken,
+  enabledQuery: store.enabledQuery,
+});
+
+export const ModelRun = ({ id, model }: ModelRunProps) => {
+  const routeInfo = useRouteInfo();
+  const { accessToken, enabledQuery } = useInstillStore(useShallow(selector));
+  const [outputActiveView, setOutputActiveView] =
+    React.useState<ModelOutputActiveView>("preview");
+  const modelRuns = usePaginatedModelRuns({
+    accessToken,
+    enabled: enabledQuery && routeInfo.isSuccess,
+    modelName: `namespaces/${routeInfo.data.namespaceId}/models/${routeInfo.data.resourceId}`,
+    pageSize: 1,
+    page: 0,
+    filter: `uid="${id}"`,
+    fullView: true,
+  });
+  const modelRun = React.useMemo(() => {
+    return modelRuns.data?.runs[0] || null;
+  }, [modelRuns.isSuccess, modelRuns.data]);
+  const taskInputOutput = React.useMemo(() => {
+    const taskPropName = convertTaskNameToPayloadPropName(
+      model?.task,
+    ) as string;
+
+    return {
+      input: modelRun?.taskInputs[0]
+        ? convertValuesToString(modelRun?.taskInputs[0][taskPropName])
+        : null,
+      output: modelRun?.taskOutputs[0]
+        ? modelRun?.taskOutputs[0][taskPropName]
+        : null,
+    };
+  }, [modelRun, model]);
+
+  const { fields, form } = useInstillForm(
+    model?.inputSchema || null,
+    taskInputOutput.input,
+    {
+      disabledAll: true,
+    },
+  );
+  const componentOutputFields = useComponentOutputFields({
+    mode: "demo",
+    schema: model?.outputSchema || null,
+    data: taskInputOutput.output,
+  });
+
+  return (
+    <React.Fragment>
```

Comment:
@orangecoloured Since this component only have one child, so we can safely remove Fragment